JOSESwift/Sources/MockedReviw/PRSerializer.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  MockedSerializer.swift
+//  JOSESwift
+//
+//  Created by Carol Capek on 06.02.19.
+//
+
+import Foundation
+
+struct PRSerializer {
+    func serialize(headerParameterAlgorithm: String, headerParametertyp: String, headerParameterKID: String, jws_payload: NSString, signature: Data) -> String? {
```

Comment:
Is there  a specific reason we are using a NSString over a string?

Reply:
Could we call this headerParametertype rather than headerParametertyp?

Reply:
Is there a reason for non camelcase for jws_payload?

Reply:
In the function, nil is not returned.  You may want to return nil when appropriate.